nf_core/lint/schema_lint.py

```diff
@@ -2,6 +2,22 @@
 
 import logging
 import nf_core.schema
+import jsonschema
+
+
+def remove_required_fields(schema):
+    """ Remove all required fields from a schema """
```

Comment:
There can also be "base-level" parameters which are not in groups which can be required. Then the `required` key is in the top level schema I think..

Reply:
Okay, didn't find an example for that unfortunately .... will they be in the `definitions` or outside of that?
Currently implemented it to be outside

Reply:
It's bad practice so hopefully examples will be difficult to find. You can make one by just adding a new param to a pipeline and running `nf-core schema build` - it will be added ungrouped like this by default (or just move an existing param out of a group in the builder).

And yes, they are outside of the definitions key.

Reply:
Alright, I'll try it out, thanks!

Reply:
Okay wild params outside definitions are also captured and controlled for 👌